### ISSUE TITLE
Evaluate Radiator

### ISSUE THREAD

**Issue by PRemmen** [RWTH-EBC/AixLib#183]
Evaluate how to proceed with Radiator

Advantages/Disadvantages to have it in parallel to Annex60 model (RadiatorEN442_2)?

**Comment by SebastianStinner** [comment]
The Radiator model is the core of the radiator model in FastHVAC => discussion necessary

**Comment by PRemmen** [comment]
Okay, in that case I will open a new issue

**Comment by mlauster** [comment]
Decision has been made and is taken care of in another issue, so I will close that issue
